app/views/static_page/home.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= provide :title, "home" %>
+
+
+<h1>Sample App </h1>
+<p>
+ This is the home page for the
+ <a href="http://www.railstutorial.org/">Ruby on
```

Comment:
You should install config gem, then put "http://www.railstutorial.org/" in setting.yml file

Reply:
done use link to and setting. for link